src/inmanta/app.py

```diff
@@ -620,6 +621,74 @@ def export(options: argparse.Namespace) -> None:
     summary_reporter.print_summary_and_exit(show_stack_traces=options.errors)
 
 
+def validate_logging_config_parser_config(
```

Comment:
I wonder if we should define subcommands for all components, so you can use this exactly like the process itself....